alphadia/workflow/managers/raw_file_manager.py

```diff
@@ -22,13 +22,15 @@ def __init__(
     ):
         """Contains and updates timing information for the portions of the workflow."""
         self.stats = {}  # needs to be before super().__init__ to avoid overwriting when loading
-
         super().__init__(path=path, load_from_file=load_from_file, **kwargs)
-        self.reporter.log_string(f"Initializing {self.__class__.__name__}")
-        self.reporter.log_event("initializing", {"name": f"{self.__class__.__name__}"})
 
         self._config: Config = config
 
+        # deliberately not saving the actual raw data here to avoid the saved manager file being too large
```

Comment:
This is just a comment why the dia_data goes back into `PeptideCentric` and doesn't stay here as one might expect?

Reply:
yes it is